src/Microsoft.IdentityModel.KeyVaultExtensions/KeyVaultSecurityKey.cs

```diff
@@ -90,7 +90,7 @@ public override int KeySize
                 if (!_keySize.HasValue)
                     Initialize();
 
-                return _keySize.Value;
+                return _keySize!.Value;
```

Comment:
are we sure this will never be null?

Reply:
It's more along the lines of it can be null based on what I saw on the Key Id:

public override string KeyId
        {
            get => _keyId!;
            set
            {
                if (string.IsNullOrEmpty(value))
                    throw LogHelper.LogArgumentNullException(nameof(value));
                else if (StringComparer.Ordinal.Equals(_keyId, value))
                    return;

                _keyId = value;

                // Reset the properties so they can be retrieved from Azure KeyVault the next time they are accessed.
                _keySize = null;
            }

Reply:
maybe something like this would work:
` return _keySize != null ? _keySize.Value : 0;`

Reply:
Yes I think that would work. On the other hand is there really a need for _keySize to be nullable in the first place if we expose it through non nullable property?

Reply:
I thought on this approach of setting a default KeySize but didn't want to define a default value as it might change its behavior somewhere down the call stack.